tests/integration/test_oclc_catalog_manager.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from config.loadEnv import loadEnvFile
+from managers import OCLCCatalogManager
+
+class TestOCLCCatalogManager:
+    @pytest.fixture
+    def testInstance(self):
+        loadEnvFile('sample-compose', fileString='config/local.yaml')
+        # TODO: In the future the catalog manager shouldn't take the OCLC no
+        return OCLCCatalogManager(1)
+
+    def testOCLCv2Auth(self, testInstance):
+        testInstance.getToken()
+        assert len(testInstance.token) > 10
+        assert testInstance.tokenExpiresAt is not None
+
```

Comment:
I think we can remove this test as this should be handled by the other manager/integration tests

Reply:
Done!

Reply:
Ah sorry, I didn't see this comment but thank you for taking care of this!